zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -16,6 +17,7 @@
 REPLICASET_TYPE = 'kube_replicaset'
 STATEFULSET_TYPE = 'kube_statefulset'
 DAEMONSET_TYPE = 'kube_daemonset'
+POSTGRESDB_TYPE = 'postgresql_database'
```

Comment:
For the most general case type `database` makes sense.  I would like to avoid tying it up with `spilo` as well, but currently this is the way we discover that we have the DB behind a service.